_scss/_script-styles.scss

```diff
@@ -50,6 +50,18 @@
 	white-space: nowrap;
 }
 
+.api-changes-col {
+	text-align: start;
+}
+
+.deprecations-col2 {
+	white-space: nowrap;
+}
+
+.deprecations-text {
+	margin-bottom: 2%;
+}
+
 .warn {
```

Comment:
@solita-ijunnone Pystyiskö näitä css:ssä olevia luokkia käyttämään niiden taulukon kenttien värittämiseen?

Reply:
Nyt tulee värit luokasta, siistimpää noin